charabia/src/tokenizer.rs

```diff
@@ -354,6 +385,15 @@ impl<'tb, A: AsRef<[u8]>> TokenizerBuilder<'tb, A> {
             segmenter_option: Cow::Owned(self.segmenter_option),
         }
     }
+
+    /// Build the tokenizer options consumming self.
+    ///
+    /// This method allows to drop the tokenizer builder without having to drop the options themselves.
+    pub fn into_options(mut self) -> (NormalizerOption<'tb>, SegmenterOption<'tb>) {
+        drop(self.build());
```

Comment:
unclear to me why the drop is needed.

Does `self.build()` mutate `self.normalizer_option` and `self.segmenter_option`?